pkgs/development/tools/github-changelog-generator/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, bundlerEnv, ruby, fetchFromGitHub,  makeWrapper}:
+let
+  env = bundlerEnv {
+    inherit ruby;
+    name = "github-changelog-generator-env";
+    gemdir = ./.;
+  };
+in stdenv.mkDerivation rec {
```

Comment:
This whole derivation looks redundant to me? What does it get you that just exporting the `bundlerEnv` doesn’t?

Reply:
is it better this way? and is tehre a way of auto removing bundle bundler and so on?

Reply:
Oh, good point! You want `bundlerApp`, not `bundlerEnv`.